adam-core/src/main/scala/org/bdgenomics/adam/util/MdTag.scala

```diff
@@ -89,7 +92,7 @@ object MdTag {
             }
             
             if (matchingBases.toInt == usedMatchingBases) {
-              mdTagStringOffset += matchingBases.size
+              mdTagStringOffset += matchingBases.length
```

Comment:
IntelliJ suggested it, said it is recommended for arrays and strings. Seems like the impl for string just immediately delegates to `.length`, so maybe it's just about saving one call of indirection